schematic/schemas/commands.py

```diff
@@ -114,35 +126,48 @@ def convert(
                 for warning in war:
                     logger.warning(warning)
 
+    if output_path:
+        output_jsonld = output_path
+
+    if output_jsonld is None:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", schema)
+    else:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", output_jsonld)
+
+    logger.info(
+        "By default, the JSON-LD output will be stored alongside the first "
+        f"input CSV or JSON-LD file. In this case, it will appear here: '{output_jsonld}'. "
+        "You can use the `--output_jsonld` argument to specify another file path."
+    )
+
+    if output_type in ["graph", "all"]:
+        logger.info("Export graph to pickle.")
+        output_graph = output_file_no_ext + ".pickle"
+        try:
+            export_graph(graph_data_model, output_graph)
+        except SystemExit as error:
+            raise error
+
+    if output_type == "graph":
```

Comment:
nit: This if block can be within the scope of `if output_type in ["graph", "all"]:`